elfi/visualization/visualization.py

```diff
@@ -397,3 +397,49 @@ def plot_discrepancy(gp, parameter_names, axes=None, **kwargs):
         axes[idx].set_axis_off()
 
     return axes
+
+
+def plot_gp(gp, parameter_names, axes=None, resol=50, const=None, **kwargs):
+    """Plot pairwise relationships as a matrix with parameters vs. discrepancy.
+
+    Parameters
+    ----------
+    gp : GPyRegression, required
+    parameter_names : list, required
+        Parameter names in format ['mu_0', 'mu_1', ..]
+    axes : plt.Axes or arraylike of plt.Axes
+
+    Returns
+    -------
+    axes : np.array of plt.Axes
+
+    """
+    n_plots = gp.input_dim
+    shape = (n_plots, n_plots)
+    axes, kwargs = _create_axes(axes, shape, **kwargs)
+
+    x_evidence = gp.X
+    y_evidence = gp.Y
+    if not const and const != 0:
+        const = x_evidence[np.argmin(y_evidence), :]
+    bounds = kwargs.get('bounds', None) or gp.bounds
+
+    for ix in range(n_plots):
+        for jy in range(n_plots):
+            if ix == jy:
+                axes[jy, ix].scatter(x_evidence[:, ix], y_evidence)
+                axes[jy, ix].set_xlim(bounds[ix])
+                axes[jy, ix].set_ylabel('Discrepancy')
+            else:
+                x1 = np.linspace(min(bounds[ix]), max(bounds[ix]), resol)
```

Comment:
Yes, in first case I was thinking to make it like this.
In my opinion, it is more evident and readable to keep `min` and `max` functions because it allows not to go into `linspace` definition and read what it is doing.

Reply:
Redundant, but ok.